client/app/pages/queries/QueryView.jsx

```diff
@@ -122,10 +123,10 @@ function QueryView(props) {
             queryFlags.canEdit &&
             !addingDescription &&
             !fullscreen && (
-              <a className="label label-tag hidden-xs" role="none" onClick={() => setAddingDescription(true)}>
+              <PlainButton className="label label-tag hidden-xs" role="none" onClick={() => setAddingDescription(true)}>
```

Comment:
This also feels heavily styled. I'm not sure that heavily styled items should be wrapped in a `PlainButton` -- it feels like maybe just adding a `role` is appropriate?

Reply:
We end in the same discussion as [here](https://github.com/getredash/redash/pull/5433#discussion_r607374939)